src/interface.jl

```diff
@@ -423,6 +423,7 @@ When `fallback` is true, and the obj does not have an extent,
 an extent is calculated from the coordinates of all geometries in `obj`.
 """
 function extent(obj; fallback=true)
+    isnothing(trait(obj)) && return Extents.extent(obj)
     ex = extent(trait(obj), obj)
     isnothing(ex) && fallback && return calc_extent(trait(obj), obj)
```

Comment:
We call `trait(obj)` 3 times on 3 consecutive lines. Perhaps better to assign it to a variable once.

Reply:
It compules away ;)

But yeah thas cleaner